authorizers/iconik/body-authorizer.ts

```diff
@@ -25,64 +26,23 @@ type Payload = CustomActionPayload & { auth_token?: string };
 export async function authorizeWithIconikCustomAction(customActionBody: Payload): Promise<IconikContext> {
   const { user_id: callerId, system_domain_id: systemDomainId, auth_token: authToken } = customActionBody;
 
-  const iconikUrl = getEnv('ICONIK_URL');
-  const appId = getEnv('ICONIK_APP_ID');
-  const requiredIconikDomainId = getEnv('ICONIK_DOMAIN_ID');
-

```

Comment:
It's better to keep these variables because this is harder to read
![image](https://user-images.githubusercontent.com/26054907/116521813-1e07d280-a8dd-11eb-8caa-6e34f1515cbf.png)